srcpkgs/rdma-core/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'rdma-core'
+pkgname=rdma-core
+version=22.1
+revision=1
+build_style=cmake
+hostmakedepends="cmake gcc libnl3-devel eudev-libudev-devel make pkg-config python3-devel valgrind-devel"
```

Comment:
remove the following

- cmake (it is automatically added by having build_style=cmake)
- gcc (part of base-devel so always included)
- make ((part of base-devel so always included)
- pkg-config (replace it with pkgconf)

Move the following to makedepends= 

- libnl3-devel
- eudev-libudev-devel
- python3-devel
- valgrind-devel

Reply:
Thank you, this is my first time using Git and xbps templates.

New commit edited.